src/index.css

```diff
@@ -0,0 +1,66 @@
+@import url('https://fonts.googleapis.com/css2?family=Poppins:wght@400;500;700&display=swap');
+
+*{
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+body{
+    background-color: #0e0e0e;
+    color: blanchedalmond;
+    font-family: sans-serif, 'Popins';
+}
+
+.navbar{

```

Comment:
:x: Please make sure there's a space in each css element as it is causing linter errors